src/components/Wallet/WalletActionsPanel.jsx

```diff
@@ -36,6 +38,33 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const Button = withStyles({
+  root: {
+    "&.Mui-disabled": {
+      pointerEvents: "auto",
+    },
+  },
+})(MuiButton);
+
+const ButtonWithTooltip = ({ tooltipText, disabled, onClick, ...other }) => {
```

Comment:
Nice! I feel like we should have a `ui` directory for these kinds of general purpose elements. We're basically just keeping these in the main `components` directory right now. I would recommend this be its own standalone component and put it there. Can be during a later refactor PR